index.html

```diff
@@ -1,325 +1,312 @@
-               </li>               
-               <li class="about-icons-li">
-                  <a href="#" class="about-icons-li-a"><img src="/assets/linkedin.svg.svg" alt="github icon"></a>
-               </li>                             
-               <li class="about-icons-li">
-                  <a href="#" class="about-icons-li-a"><img src="/assets/medium.svg.svg" alt="github icon"></a>
-               </li>
-            </ul>
-         </div>
-         <ul class="about-tools-list">
-            <li class="about-tool">
-               <div class="about-languages-list">
-                  <h3 class="about-lang-title">
-                     Languages
-                  </h3>
-                  <img class="about-lang-img" src="./assets/chev-down.png" alt="arrow icon">
-               </div>
-               <ul class="about-lang">
-                  <li class="about-lang-row">
-                     <img src="./assets/javascript.svg.svg" alt="javascript icon">
-                     <p class="about-lang-name">
-                        JavaScript
-                     </p>
-                  </li>
-                  <li class="about-lang-row">
-                     <img src="./assets/html.svg.svg" alt="javascript icon">
-                     <p class="about-lang-name">
-                        HTML
-                     </p>
-                  </li>
-                  <li class="about-lang-row">
-                     <img src="./assets/css.svg.svg" alt="javascript icon">
-                     <p class="about-lang-name">
-                        CSS
-                     </p>
-                  </li>
-               </ul>
+            <li>
+              <input type="email" name="email" placeholder="Enter your email" required />
             </li>
-            <li class="about-tool border-bottom">
-               <div class="about-languages-list">
-                  <h3 class="about-lang-title">
-                     Frameworks
-                  </h3>
-                  <img class="about-lang-img" src="./assets/chev-right.png" alt="arrow icon">
-               </div>
+            <li>
+              <textarea name="message" placeholder="Write your message here" required maxlength="500">
+              </textarea>
             </li>
-            <li class="about-tool border-bottom">
-               <div class="about-languages-list">
-                  <h3 class="about-lang-title">
-                     Skills
-                  </h3>
-                  <img class="about-lang-img" src="./assets/chev-right.png" alt="arrow icon">
-               </div>
+            <li>
+              <button type="submit">Get It Touch</button>
             </li>
-         </ul>
-      </section>
-   </body>
-</html>
\ No newline at end of file
+          </ul>
+        </form>
+        <span><img src="./Shape.png" alt="image" /></span>
+      </div>
+    </section>
```

Comment:
- [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is missing a background vector image from the background, Please place the background image and make sure it matches the design.

Your design

![Screenshot (171)](https://user-images.githubusercontent.com/83803414/155211436-f5a4bf83-6b56-460f-84bf-e49de6502bbd.png)

Figma design

![Screenshot (170)](https://user-images.githubusercontent.com/83803414/155211457-9f30f536-f1b2-47f7-83eb-4812d7c55485.png)